### PR TITLE
Refactored doc

### PR DESCRIPTION
**Pull request by john-0x**
Ignore risk-management.  will need to move this elsewhere in the structure. the five individual risk management policy contracts are documented in Policies.

### CODE REVIEW

File: chapters/fund/accounting.md [hunk 1]
```diff
@@ -127,12 +137,9 @@ This function maintains the `ownedAssets` array and the `isInOwedAssets` mapping
 
 `function addAssetToOwnedAssets(address _asset) public auth`
 
-This function maintains the `ownedAssets` array and the `isInOwedAssets` mapping by adding asset addresses as the fund holdings composition changes.
+This function requires that the caller is the `owner` or the current contract. The function maintains the `ownedAssets` array and the `isInOwedAssets` mapping by adding asset addresses as the fund holdings composition changes.
```
Comment (GauthamN) [inline chapters/fund/accounting.md:140]: Maybe add more who the called is supposed to be i.e adapter (while msg.sender is the current contract)

File: chapters/fund/participation.md [hunk 2]
```diff
@@ -2,59 +2,102 @@
...
 
-Public Functions
+`function disableInvestment(address[] _assets) public auth`
 
-`function requestInvestment(uint requestedShares, uint investmentAmount, address investmentAsset) external`
+This function requires that the caller is the `owner` or the current contract. The function iterates over an array of provided asset token addresses and ensures that asset token addresses are set to `false` in the `investAllowed` mapping.
 
-This function ensures that the fund is not shutdown, then populates the `Request` struct and adds this to the `requests` mapping corresponding to the `msg.sender`.  This function also sets `SHARE_DECIMALS` to 18.
+
+`function requestInvestment(uint requestedShares, uint investmentAmount, address investmentAsset) external payable amguPayable`
```
Comment (GauthamN) [inline chapters/fund/participation.md:90]: Should we mention reasoning behind two step process here?

Comment (jennazenk) [reply chapters/fund/participation.md:90]: agree opening an issue for this

File: chapters/fund/accounting.md (hunk 1 above)
Comment (jennazenk) [reply chapters/fund/accounting.md:140]: you can make the change directly @GauthamN

**Pull request merged**